python/tvm/tir/expr.py

```diff
@@ -1093,20 +1093,27 @@ class BufferLoad(PrimExprWithOp):
         The buffer to be loaded.
 
     indices : List[PrimExpr]
-        The buffer indices.
+        The buffer indices to load values from.
 
     span : Optional[Span]
         The location of this expression in the source code.
+
+    predicate : Optional[PrimExpr]
+        A vector mask of int1 values indicating which lanes of a vector are to be loaded.
```

Comment:
At first glance, the `int1` reads like a typo.  Can we explicitly call it `boolean values` instead?

Reply:
Done. There was a bit of confusion here since LLVM intrinsics expect predicate masks of `int1` values. However, boolean types are already converted to `int1` in the codegen backend. So I have required `uint1`/boolean types in TVM.